docs/user/redis/usage.rst

```diff
@@ -10,8 +10,10 @@ Parameter                              Type    Description
 ====================================== ======= ====================================================================================
 ``redis.url *``                        String  Redis connection URL. The URL can be used to specify the Redis database and
                                                credentials, if required - for example, 'redis://user:pass@host:port/db'
+                                               If using a Redis cluster, this may be a comma-separated list of redis URLs.
```

Comment:
i think these get rendered into one line (and then re-split based on browser width), so we'd want a period after the last sentence:

```suggestion
                                               credentials, if required - for example, 'redis://user:pass@host:port/db'.
                                               If using a Redis cluster, this may be a comma-separated list of redis URLs.
```